src/core/namespace.c

```diff
@@ -1170,14 +1170,16 @@ static int mount_procfs(const MountEntry *m, const NamespaceInfo *ns_info) {
                          * user namespace. */
                         r = mount_nofollow_verbose(LOG_DEBUG, "/proc", entry_path, NULL, MS_BIND|MS_REC, NULL);
                         if (r < 0)
-                                return -EPERM;
+                                return r;
                 }
+                return 1;
```

Comment:
Does `1` have any meaning here? IIUC, this function is only called by `apply_one_mount`, which is only called by `apply_mounts`, which only checks for `r < 0`.

Reply:
I think we can change 1 → 0.

Reply:
OK. One more commit is added about the suggestion.